File: Challenges/Problem_2/Even_Fibonacci_Numbers.py
# KuntaNgcobo
# Even Fibonacci Numbers

def fiboEvenSum(n):
  tempEvens = 2
  holderFib = [1, 2]
  
  for i in range(2, n):
    temp = holderFib[1]
    holderFib[1] = holderFib[0] + temp
    holderFib[0] = temp

    tempEvens += holderFib[1] if(holderFib[1]%2 == 0) else 0

  return tempEvens, holderFib;

# print(fiboEvenSum(10))

File: Challenges/Problem_0002/Even_Fibonacci_Numbers.js
// KuntaNgcobo
// Even Fibonacci Numbers

const fiboEvenSum = n => {
  let tempEvens = 2;
  let holderFib = [1, 2];
  
  for(var i = 2; i < n; i++){
    let temp = holderFib[1];
    holderFib[1] = holderFib[0] + temp;
    holderFib[0] = temp;

    tempEvens += holderFib[1]%2 === 0 ? holderFib[1] : 0;
  }
  return tempEvens;
}